assets/js/components/setup/ModuleSetup.js

```diff
@@ -65,9 +67,7 @@ export default function ModuleSetup( { moduleSlug } ) {
 			redirectURL = getSiteKitAdminURL( 'googlesitekit-dashboard', args );
 		}
 
-		delay( function() {
-			global.location.replace( redirectURL );
-		}, 500, 'later' );
+		navigateTo( redirectURL );
```

Comment:
We need to rely on this one in a couple more components. The original specific goal of the issue was to fix the scopes notification from showing up when activating a module before getting redirected to OAuth. In order to satisfy that criteria, we'll need to update the `assets/js/components/settings/SetupModule.js` file accordingly.

There are various other files where we redirect to the OAuth flow, all of which would be worth adjusting, to fix the same problem:
* `ActivateModuleCTA`
* `CompleteModuleActivationCTA`
* `PermissionsModal`

Or, even better, we could replace all usages of `global.location.assign` with the `navigateTo` action to set the right precedent also for the future.

Reply:
Yep, updated all usages of `global.location.assign` with the `navigateTo`.